src/main/kotlin/caculator/Number.kt

```diff
@@ -0,0 +1,37 @@
+package caculator
+
+internal class Number(val value: Int) {
```

Comment:
* Number에 대한 단위 테스트를 해보면 어떨까요?
* Number가 데이터 클래스이면 어떨까요?
* 주 생성자가 내부에서만 쓰인다면 private으로 선언해주세요!
```suggestion
internal class Number private constructor(val value: Int) {
```

Reply:
주로 부생성자를 통해 생성되지만,
실제 주생성자도 사용할수 있다고 생각해서 제한하지 않았습니다.

Reply:
데이터 클래스의 용도를 알수있을까요. 제가 생각하기엔 vo, dto 에 사용될거같습니다.

Reply:
넵! 저도 흔히 dto,vo라고 불리는 객체들에게 data class를 적용할 수 있다고 생각하는데요!!
주영님은 어떤 이유로 data class가 dto,vo에 적합하다고 생각하셨을까요~?

vo와 관련된 [이 글](https://woowacourse.github.io/javable/post/2020-06-11-value-object/)이 주영님의 생각에 도움이 됐으면 좋겠습니다!

Reply:
해당글을 통해 생각이 정리되었습니다. 감사합니다.

Number 객체는 불변 + 숫자에 대한 비지니스 로직을 담당하고있어서 적합하다고 생각이 들었습니다.